imap_processing/idex/idex_packet_parser.py

```diff
@@ -90,16 +92,36 @@ def __init__(self, packet_file: str):
             dust_event.process() for dust_event in dust_events.values()
         ]
 
-        self.data = xr.concat(processed_dust_impact_list, dim="Epoch")
+        self.l1_data = xr.concat(processed_dust_impact_list, dim="Epoch")
 
 
 class RawDustEvent:
+    """
+    This class encapsulates the work needed to convert a single dust event into a
+    processed XArray Dateset object
+    """
+
     # Constants
-    HIGH_SAMPLE_RATE = 1 / 260  # nanoseconds per sample
-    LOW_SAMPLE_RATE = 1 / 4.0625  # nanoseconds per sample
+    HIGH_SAMPLE_RATE = 1 / 260  # microseconds per sample
+    LOW_SAMPLE_RATE = 1 / 4.0625  # microseconds per sample
+
+    NUMBER_SAMPLES_PER_LOW_SAMPLE_BLOCK = (
+        8  # The number of samples in a "block" of low sample data
+    )
+    NUMBER_SAMPLES_PER_HIGH_SAMPLE_BLOCK = (
+        512  # The number of samples in a "block" of high sample data
```

Comment:
There are 8 samples in a block, where a sample is just one unit of data collection.  Some sensors in the instrument take samples every 1/4.0625 microseconds, so a block of this low sample data is about 2 microseconds.  The only time that a block of data matters is in a function below, where we calculate how many samples of data are included before the before the dust impact triggered the instrument. 

For whatever reason, instead of the number of samples before triggering, the header has information about the number of blocks before triggering.